README.md

```diff
@@ -20,11 +20,18 @@ In order to interact with any package contained in this repository you will need
 
 - [**NVM**](https://github.com/nvm-sh/nvm/blob/master/README.md)
 - [**Homebrew**](https://brew.sh/)
-- [**Node.js@18.x.x**](https://nodejs.org/)
+- [**Node.js@16.x.x**](https://nodejs.org/) `16.x.x`
 - [**Yarn**](https://www.npmjs.com/package/yarn)
 - [**Cocoapods**](https://cocoapods.org/)
-- [**Xcode@14.2**](https://xcodereleases.com/)
-- [**Android Studio**](https://developer.android.com/studio)
+- [**Xcode**](https://developer.apple.com/xcode/) `14.2`
+  - [Download older versions of Xcode here](https://developer.apple.com/download/all/?q=xcode%2014.2)
+- [**Android Studio**](https://developer.android.com/studio) `Chipmunk`
+  - [Download older versions of Android Studio here](https://developer.android.com/studio/archive)
+  - [Install a Virtual Device](https://developer.android.com/studio/run/emulator#avd) (`Nexus 5` with `UpsideDownCake` good default)
+  - When [installing the Android SDK](https://reactnative.dev/docs/environment-setup#android-sdk) use these:
+  - `Android 12.0`
```

Comment:
We can bump this to 13 now too if we want. 12 should still work fine though if you want to leave it. 12 is probably more common out there anyway

Reply:
I can note that, Andoird 12 was the latest supported by the Chipmunk version of Android Studio that I could find